packages/frameworks/frameworks.json

```diff
@@ -4,70 +4,329 @@
     "slug": "nextjs",
     "logo": "https://raw.githubusercontent.com/zeit/now/master/packages/frameworks/logos/next.svg",
     "tagline": "Next.js makes you productive with React instantly — whether you want to build static or dynamic sites. ",
-    "website": "https://nextjs.org"
+    "website": "https://nextjs.org",
+    "detectors": {
+      "every": [
+        {
+          "file": "package.json",
+          "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"next\":\\s*\".+?\"[^}]*}"
```

Comment:
This seems very verbose and prone to error.

Is there anyway we could minimize the amount of copy/pasta for each framework?

Reply:
We tried to make it as generic as possible

cc @leo